docs/docs/dev_docs/getting_started/updating.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Updating
+---
+
+There are 4 components whose versions need to be kept compatible:
+1. Aztec Sandbox,
+2. Aztec CLI,
+3. Noir compiler `nargo`,
+4. Noir framework for Aztec contracts `aztec.nr`.
```

Comment:
I'd include a short paragraph after this explaining that "the versions of Sandbox, CLI, and aztec.nr should all match, and the `nargo` version should be the one reported by the `get-node-info` CLI command", as a TLDR.

Reply:
Now that I think of it, we could also show the compatible nargo version in the sandbox when it starts up, next to its own version.

Reply:
> I'd include a short paragraph after this explaining that "the versions of Sandbox, CLI, and aztec.nr should all match, and the nargo version should be the one reported by the get-node-info CLI command", as a TLDR.

Good idea. Addressed in e783331.

Reply:
> Now that I think of it, we could also show the compatible nargo version in the sandbox when it starts up, next to its own version.

Addressed in [e3e73ff](https://github.com/AztecProtocol/aztec-packages/pull/2297/commits/e3e73ff776342d43316584323bb8ad5980dc4edf)